src/js/tabs/trade.controller.js

```diff
@@ -834,7 +836,7 @@ TradeTab.prototype.angular = function(module)
       if (order.price_amount && order.price_amount.is_valid() &&
           order.first_amount && order.first_amount.is_valid()) {
         order.second_amount = order.price_amount.product_human(order.first_amount);
-        order.second = +order.second_amount.to_human({group_sep: false});
+        order.second = order.second_amount.to_human({group_sep: false});
```

Comment:
@jublo That was whole point of this commit. This value goes into input field. And when it is number, chrome shows it in e-notation if it too small or too big